### PR TITLE
synchronize help text to actual option name

### PR DESCRIPTION
**Pull request by t-8ch**

### CODE REVIEW

**Comment by falconindy** [comment]
Nice catch. This also needs fixing in the manpage.

**Comment by t-8ch** [comment]
@falconindy  and the completions...
So I'll change it the other way round, so the code matches all of the documentation

File: ponymix.cc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include "pulse.h"
+ "pulse.h"
```
Comment (falconindy) [inline ponymix.cc]: that can't be right...

Comment (t-8ch) [inline ponymix.cc]: I fat fingered my vim. Now it is correct...

**Comment by falconindy** [comment]
Thanks, merged manually.

**Pull request closed without merging**